docs/deploy/hardening-guide.md

```diff
@@ -0,0 +1,112 @@
+# Hardening Guide
+
+## Overview
+There are several ways to do hardening and securing of nginx. In this documentation two guides are used, the guides are 
+overlapping in some points:
+
+- [nginx CIS Benchmark](https://www.cisecurity.org/benchmark/nginx/)
+- [cipherlist.eu](https://cipherlist.eu/) (one of many forks of the now dead project cipherli.st)
+
+This guide describes, what of the different configurations described in those guides is already implemented as default 
+in the nginx implementation of kubernetes ingress, what needs to be configured, what is obsolete due to the fact that
+the nginx is running as container (the CIS benchmark relates to a non-containerized installation) and what is difficult
+or not possible.
+
+Be aware that this is only a guide and you are responsible for your own implementation. Some of the configurations may
+lead to have specific clients unable to reach your site or similar consequences.
+
+This guide refers to chapters in the CIS Benchmark. For full explanation you should refer to the benchmark document itself
+
+## Configuration Guide
+| Chapter in CIS benchmark | Status | Default | Action to do if not default|
+|:-------------------------|:-------|:--------|:---------------------------|
+| __1 Initial Setup__ ||| |
+| ||| |
+| __1.1 Installation__||| |
+| 1.1.1 Ensure NGINX is installed (Scored)| OK | done through helm charts / following documentation to deploy nginx ingress | |
+| 1.1.2 Ensure NGINX is installed from source (Not Scored)| OK | done through helm charts / following documentation to deploy nginx ingress | |
+| ||| |
+| __1.2 Configure Software Updates__||| |
+| 1.2.1 Ensure package manager repositories are properly configured (Not Scored) | OK | done via helm, nginx version could be overwritten, however compability is not ensured then| |
+| 1.2.2 Ensure the latest software package is installed (Not Scored)| ACTION NEEDED | done via helm, nginx version could be overwritten, however compability is not ensured then| Plan for periodic updates |
+| ||| |
+| __2 Basic Configuration__ ||| |
+| ||| |
+| __2.1 Minimize NGINX Modules__||| |
+| 2.1.1 Ensure only required modules are installed (Not Scored) | ??? | ??? | |
```

Comment:
this is already being done? Any idea/proposal to reduce the number of dependencies is welcome

Reply:
Hm, well, you are right. I added information reflecting that.